bird_Docker_build/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9
+
+WORKDIR /
+
+RUN wget https://github.com/stan-dev/cmdstan/releases/download/v2.30.0/cmdstan-2.30.0.tar.gz
+RUN tar -zxpf cmdstan-2.30.0.tar.gz
+
+RUN mv cmdstan-2.30.0 cmdstan
+WORKDIR /cmdstan
+RUN make build
+
+RUN wget --no-check-certificate https://github.com/bmajoros/BIRD/blob/master/bird.py
+
+COPY BIRD.stan .
+RUN make BIRD
+RUN mv BIRD BIRD-1.1
+
+ENTRYPOINT ["bird.py", "BIRD-1.1"]  
```

Comment:
I think this should be a two-stage file -- the first stage builds BIRD, and the second stage runs BIRD, something like this:

```Dockerfile
FROM python:3.9 as BIRD

WORKDIR /

RUN wget https://github.com/stan-dev/cmdstan/releases/download/v2.30.0/cmdstan-2.30.0.tar.gz
RUN tar -zxpf cmdstan-2.30.0.tar.gz

RUN mv cmdstan-2.30.0 cmdstan
WORKDIR /cmdstan
RUN make build
COPY BIRD.stan .
RUN make BIRD


FROM python:3.9

WORKDIR /
RUN git clone https://github.com/tncowart/python
ENV PYTHONPATH "${PYTHONPATH}:/python"

WORKDIR /bird
RUN pip3 install pypi.org pandas
COPY --from=BIRD /cmdstan/BIRD .
RUN wget https://github.com/bmajoros/BIRD/blob/master/bird.py

ENTRYPOINT ["python", "bird.py", "BIRD"] 
```

I'm not sure that the above works, but I think that's roughly how this Dockerfile should be structured.